examples/lujain_work_area/Contact_Book/contact_book.py

```diff
@@ -0,0 +1,27 @@
+# contact_book.py
+
+from contact_records import ContactRecords
+
+class ContactBook:
+    def __init__(self, csv_file):
+        self.contact_records = ContactRecords(csv_file)
+
+    def add_contact(self, name, phone, email, address):
+        contact = {'Name': name, 'Phone': phone, 'Email': email, 'Address': address}
+        self.contact_records.add_contact(contact)
+
+    def delete_contact(self, identifier):
+        self.contact_records.delete_contact(identifier)
```

Comment:
logically we should be giving it a ContactRecord and this method can handle the delete logic based on a unique attribute of ContactRecord object